File: matlab/affine_pol_data.m
% affine policy.
m = 20; n = 10; p = 5;

randn('state',0); rand('state',0);
A = randn(m,n);
c = rand(n,1);
b0 = ones(m,1);
B = .15*sprandn(m,p,.3);

File: python/affine_pol_data.py
# affine policy.

import numpy as np
import scipy.sparse

np.random.seed(0)

m, n, p = 20, 10, 5

A = np.random.randn(m, n)
c = np.random.rand(n, 1)
b0 = np.ones((m, 1))
B = .15 * scipy.sparse.rand(m, p, .3)
